README.md

```diff
@@ -254,6 +254,12 @@ ComfyJS.Disconnect();
  - **onHosted**`( user, viewers, autohost, extra )`
     - Responds to channel being hosted
     - Requires being authorized as the broadcaster
+ - **onBan**`( bannedUsername, reason, extra )`
+    - Responds to a user being banned
+    - reason will be null unless authorized as broadcaster or moderator for the channel
```

Comment:
According to <https://github.com/tmijs/docs/blob/gh-pages/_posts/v1.4.2/2019-03-03-Events.md> linked in #23, `reason` is deprecated and should _always_ be `null` - is this still the case?